src/Wpf/Prism.Wpf/Services/Dialogs/IDialogResult.cs

```diff
@@ -1,9 +1,13 @@
 ﻿namespace Prism.Services.Dialogs
 {
+    /// <summary>
+    /// Contains <see cref="IDialogParameters"/> passed to the dialog
+    /// and the <see cref="ButtonResult"/> of the dialog.
+    /// </summary>
     public interface IDialogResult
     {
         /// <summary>
-        /// The parameters from the dialog
+        /// The parameters passed to the dialog.
```

Comment:
I believe this was correct to begin with... The DialogResult should be providing the DialogParameters from the Dialog... not to it..

Reply:
No problem, I thought the `Parameters` property in `IDialogResult` was the original parameters passed to the Dialog (IE `IDialogAware.OnDialogOpened(parameters)`)